### PR TITLE
Working with APIs

### PR DESCRIPTION
**Pull request by ArifPMan**

### CODE REVIEW

**Comment by ArifPMan** [comment]
Please review and provide your feedback comments.

**Comment by ArifPMan** [comment]
Hello, Vineet,

I have addressed most of the comments except versioning and Generating collections from schema. Could you please review this? I am working on the rest of the things.

File: v6/postman/working_with_APIs/sharing_apis.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: "Sharing apis and managing roles"
+page_id: "sharing_apis"
+warning: false
+---
+
+You can easily share your APIs with other users and teams on Postman - both in the app and in the [web dashboard](https://app.getpostman.com/dashboard). In Postman, you generally share something to a workspace. To share an API or a collection in Postman, you first need to share the workspace in which your API/Collection is present and then assign individual permissions to your team members. You can then designate view (Viewer) or edit (Editor) permissions to the team/members.
```
Comment (preethammavin) [inline v6/postman/working_with_APIs/sharing_apis.md]: @ArifPMan other users and teams does reflect the right intention. I am sharing with my team

Comment (preethammavin) [inline v6/postman/working_with_APIs/sharing_apis.md]: In Postman you share your collections or APIs to workspaces. To share an API with your team, you need to share the API to a shared workspace or share the workspace itself with the team.

File: v6/postman/working_with_APIs/sharing_apis.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+Let's say you created an API, added a collection as documentation and added another collection as a test suite. Now, you want to share this API to a workspace member M1 who is in a team workspace T1. When you share this API to workspace T1, you don't really share your collections with the workspace member M1. You must share those collections (documentation, test suites etc) individually again in order for them to access. Remember, it's a two-step process. When you delete or remove the API from the workspace T1, these entities do not get removed and the workspace member M1 can still view these collections. You must remove them explicitly.
```
Comment (preethammavin) [inline v6/postman/working_with_APIs/sharing_apis.md]: I would tell the user that when you share the API with your team, they have access to see the listing of the elements attached to the API but not the elements itself unless they are also shared with the team

File: v6/postman/working_with_APIs/the_api_workflow.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+In the **Test** tab, click **Add integration tests** to check if your endpoints work as expected when combined with other APIs or services. The following screen opens:
+
+[![api add integration test](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/API-Add-Integration-Test1.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/API-Add-Integration-Test1.png)
+
+In the above screen, select a corresponding collection from the list and click **Add Integration Test**. The integration tests attached to the collection are now added to the API. Only available collections will be shown in this list.
+
+#### Adding a contract test
+
+If you're writing tests to evaluate the integration of your API with other services, you can use **Contract tests**. In the **Test** tab, click **Add contract tests** to ensure the API works according to its specification. The following screen opens:
```
Comment (preethammavin) [inline v6/postman/working_with_APIs/the_api_workflow.md]: I wouldnt use integration as a word here as that would be redundant between contract test and integration test

File: v6/postman/working_with_APIs/versioning_an_api.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
+---
+title: "Versioning APIs"
+page_id: "versioning_an_api"
+warning: false
+---
+
+Postman allows you to manage multiple versions of your APIs. You can link your collections to specific versions of your API by adding version tags to them. This enables you to maintain versioning of your collection revisions with a corresponding version of your API. 
+
+Though Postman allows you to tag versions to your collections, it however, does not allow you to manually handle collection version tags. Postman allows explicit versioning only to APIs. You will also be able to create separate monitors, mocks, public documentation and run on all tagged versions of your collections.
```
Comment (preethammavin) [inline v6/postman/working_with_APIs/versioning_an_api.md]: what would manually handle mean? might not represent it that way

also would not include "run" for the actions which can be done on version tag

**Pull request merged**